backend/app/api/endpoints/knowledge.py

```
@@ -1629,140 +1601,709 @@ def _update_kb_summary_after_deletion(kb_id: int, user_id: int, user_name: str):
+    )
+
+
+# ============== Permission Request Endpoints ==============
+
+permission_request_router = APIRouter()
+
+
+@permission_request_router.post("", response_model=PermissionRequestResponse)
+async def create_permission_request(
+    data: PermissionRequestCreate,
+    current_user: User = Depends(security.get_current_user),
```

Ruff (0.14.14)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)